frontend/src/metabase/visualizations/visualizations/PivotTable.jsx

```diff
@@ -17,17 +18,17 @@ const partitions = [
   {
     name: "rows",
     columnFilter: isDimension,
-    title: t`Fields to use for the table rows`,
+    title: t`Row fields`,
```

Comment:
It's too late for this I think since we've already sent strings to the translators. Sorry I went ahead with the prototype strings without double-checking.